proto/osmosis/lockup/lock.proto

```diff
@@ -55,3 +55,14 @@ message QueryCondition {
     (gogoproto.moretags) = "yaml:\"timestamp\""
   ];
 }
+
+// SyntheticLock is a single unit of synthetic lockup
+message SyntheticLock {
+  uint64 lock_id = 1;
+  string suffix = 2;
+  google.protobuf.Timestamp end_time = 3 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"end_time\""
+  ];
```

Comment:
Shouldn't this be duration?

Reply:
Synthetic lockup is managed by external module and synthetic lockup's duration is different from native lockup.
Here end time is when synthetic lock up is deleted automatically from the store. And this end time is set on superfluid module as current time + 3 weeks unbonding time - users could try to do superfluid staking for his 100 year lockup.

Reply:
I think this needs more comments here to explain it then.

Reply:
If its supposed to be the case that end time is optional, can there be a boolean for "active"

Reply:
For active one, end_time is set to time.Time{} - this kind of mechanism is working on native lockups as well. Yeah here end time is only set for unbonding synthetic lockups. For delegated lockups, it's bonding synthetic lockups.
I will add a bit of comments for end_time.

Reply:
Hrmm, seems ok for now then, but we should consider changing to be different structs prior to it going live. Lets file this into an issue pre-merge.